sigal/templates/sigal.conf.py

```diff
@@ -40,6 +40,9 @@
 # Size of resized image (default: (640, 480))
 img_size = (800, 600)
 
+# Output format of images (default: None, i.e. use input format)
+img_format = "JPEG"
+
```

Comment:
This should be commented as this file is meant to be used with the default values when creating a new gallery. Could you also add in the comment a list of possible values for the most common formats (e.g. png) ?